toolkit/scripts/toolchain/cross/gcc_cross.sh

```diff
@@ -0,0 +1,86 @@
+# Docs used to create this script:
```

Comment:
Please include standard Microsoft OSS header and license info at the top

Reply:
Done